pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -1456,6 +1460,27 @@ type PrivateCluster struct {
 	AdditionalEndpointServices []string `json:"additionalEndpointServices,omitempty"`
 }
 
+// InstanceSelector holds EC2 instance selector options
+type InstanceSelector struct {
+	// VCPUs specifies the number of vCPUs
+	VCPUs int `json:"vCPUs,omitempty"`
+	// Memory specifies the memory in GiB
+	Memory string `json:"memory,omitempty"`
```

Comment:
does the memory type need to be a `string` if the unit is predetermined? i can see users assuming it is `int` like the others and getting an error

Reply:
That comment is not accurate, the unit defaults to GiB but can be set to MiB etc. I'll update the comment.

Reply:
aah ok makes sense 👍